### PR TITLE
[RFC] Implement issue #40: Tab completion

### PR DESCRIPTION
**Pull request by BlackEdder**
Enable tab completion for transactions.

### CODE REVIEW

File: lib/reckon/app.rb [hunk 1]
```diff
@@ -115,7 +119,8 @@ def output(ledger_line)
       options[:output_file].flush
     end
 
-    def guess_account(row)
+    # Weigh accounts by how well they match the row
+    def weighted_account_match( row )
```
Comment (cantino) [inline lib/reckon/app.rb]: Looks like a didn't write a spec about this method.  Oops :)  Up to you if you want to backfill one, it's okay with me if you don't.

**Comment by cantino** [comment]
Nice!

**Comment by BlackEdder** [comment]
Did some refactoring and readded guess_account, because pull request #47 depends on it. Also added a simple test for guess_account.

**Comment by cantino** [comment]
Looks great to me!

**Pull request merged**